src/components/Cast/TextareaInfo.tsx

```diff
@@ -0,0 +1,78 @@
+import { AccentText } from 'components/ui/Text'
+import { displayFrom, displayTo } from 'helpers/visibilityClassnames'
+import Button from 'components/ui/Button'
+import CharInCircle from 'components/ui/CharInCircle'
+import Sizes from 'models/MarkSizes'
+import Tooltip from 'components/ui/Tooltip'
+import classnames, {
+  alignItems,
+  alignSelf,
+  display,
+  flexDirection,
+  fontSize,
+  gap,
+  justifyContent,
+  margin,
+  space,
+  textDecoration,
+} from 'classnames/tailwind'
+
+const hintWrapper = classnames(
+  display('inline-flex'),
+  alignItems('items-center'),
+  gap('gap-x-1'),
+  fontSize('text-xs')
+)
+
+const textAreaInfoWrapper = classnames(
+  display('flex'),
+  justifyContent('justify-between'),
+  flexDirection('md:flex-row', 'flex-col'),
```

Comment:
Tailwind is mobile first, can you please put mobile styles first and `breakpoints:` styles second everywhere?

Reply:
I think I fixed it everywhere, ty